spec/SensioLabs/Behat/PageObjectExtension/PageObject/Factory/CamelcasedClassNameResolverSpec.php

```diff
@@ -77,4 +86,22 @@ function it_throws_an_exception_if_element_class_does_not_exist()
         $this->shouldThrow(new \InvalidArgumentException($expectedMessage))
             ->duringResolveElement('Recent searches');
     }
+
+    function it_resolves_an_element_class_given_as_fqcn()
+    {
+        $this->beConstructedWith(array('\\Foo', '\\Element'));
+
+        $elementFQCN = 'spec\\SensioLabs\\Behat\\PageObjectExtension\\PageObject\\Factory\\Bar';
+        $this->resolvePage($elementFQCN)->shouldReturn($elementFQCN);
+    }
+}
+
+class Foo extends Page
```

Comment:
Umh, yes, but in that case I could not tell the difference in the examples, I mean, they're going to work even if FQCN part is broken. Maybe I can create brand new ones that not respects Page/Element structures in `Fixtures` folder?